emulation_configure.bash

```diff
@@ -203,6 +203,10 @@ function verify_host_environment() {
     [[ ${VERIFIED_QEMU_VERSIONS[*]} =~ ${4:0:5} ]] || die "qemu is not version" ${VERIFIED_QEMU_VERSIONS[*]}
     verify_QBH
 
+    # check if system is running apparmor
```

Comment:
I fixed the apparmor problems so this test isn't needed.  Also, I think the test will fail if apparmor is NOT loaded...